src/kubectlUtils.ts

```diff
@@ -0,0 +1,107 @@
+import * as vscode from "vscode";
+import { Kubectl } from "./kubectl";
+
+export type Cluster = {

```

Comment:
This doesn't seem like a normal use of type aliasing (I had to look up the `type` keyword to work out what it did!).  Consider making these interfaces with read-only members.